src/ui/components/Library/Team/View/TestRuns/BranchIcon.tsx

```diff
@@ -13,11 +13,7 @@ export function BranchIcon({
 }) {
   const theme = useTheme();
```

Comment:
I think we remove this line and probably the import of `useTheme` too

Reply:
Good call, fixed